test/cpp/security/tls_certificate_verifier_test.cc

```diff
@@ -209,12 +197,12 @@ TEST(TlsCertificateVerifierTest, VerifiedRootCertSubjectVerifierFailsNull) {
 
 TEST(TlsCertificateVerifierTest, VerifiedRootCertSubjectVerifierFailsMismatch) {
   grpc_tls_custom_verification_check_request request;
-  char* expected_subject = const_cast<char*>(
-      "CN=testca,O=Internet Widgits Pty Ltd,ST=Some-State,C=AU");
+  constexpr char kExpectedSubject[] =
+      "CN=testca,O=Internet Widgits Pty Ltd,ST=Some-State,C=AU";
   request.peer_info.verified_root_cert_subject = "BAD_SUBJECT";
   auto verifier =
       ExternalCertificateVerifier::Create<VerifiedRootCertSubjectVerifier>(
-          expected_subject);
+          const_cast<char*>(kExpectedSubject));
```

Comment:
Same here: `const_cast<>` should not be needed.

Reply:
Removed them